MusicTaste/MusicTaste/View Controllers/ProfileViewController.m

```diff
@@ -14,10 +13,9 @@ @implementation ProfileViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
-    //Sets username
     self.screenName.text = [[User user] name];
-    //Gets image Url
     NSString *URLString = [[User user] profilePicture];
+    //Removes the string "normal" from url to be able to use the URL
```

Comment:
Explained what I was deleting from the url

Reply:
Think about this from the perspective of someone who is unfamiliar with the Spotify API. Why can `_normal` not be part of the URL? The current comment doesn't really explain that.